README.md

```diff
@@ -46,6 +46,7 @@ Evaluated
 * [pgvector](https://github.com/pgvector/pgvector) ![https://img.shields.io/github/stars/pgvector/pgvector?style=social](https://img.shields.io/github/stars/pgvector/pgvector?style=social)
 * [RediSearch](https://github.com/redisearch/redisearch) ![https://img.shields.io/github/stars/redisearch/redisearch?style=social](https://img.shields.io/github/stars/redisearch/redisearch?style=social)
   * [pg_embedding](https://github.com/neondatabase/pg_embedding) ![https://img.shields.io/github/stars/pg_embedding/pg_embedding?style=social](https://img.shields.io/github/stars/neondatabase/pg_embedding?style=social)
+* [Descartes(01AI)](https://github.com/01-ai/Descartes.git)
```

Comment:
This link doesn't work.

Reply:
@maumueller  OK, We remove the link  and add it back after the product is released in the future.

Reply:
It wasn't my intention to remove it. I think you are pointing to the wrong github repo in the link, so please just edit.